keps/sig-cluster-lifecycle/generic/draft-20200508-communicating-a-local-registry/README.md

```diff
@@ -0,0 +1,528 @@
+<!--
```

Comment:
hi, thanks for sending this!
the path should be `sig-cluster-lifecycle/generic/<tracking-issue-#>-communicating-a-local-registry/...`

Reply:
done!